src/ui/users/list/UserListTable.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Col, Paginator, Alert } from 'patternfly-react';
+import { FormattedMessage } from 'react-intl';
+import { formattedText } from 'frontend-common-components';
+import UserListMenuActions from 'ui/users/list/UserListMenuActions';
+import UserStatus from 'ui/users/common/UserStatus';
+
+class UserListTable extends Component {
+  constructor(props) {
+    super(props);
+
+    this.changePage = this.changePage.bind(this);
+  }
+
+  componentWillMount() {
+    this.props.onWillMount();
+  }
+
+  changePage(page) {
+    this.props.onWillMount(page);
+  }
+
+  renderTableRows() {
+    return this.props.users.map(user => (
+      <tr key={user.username}>
+        <td className="UserListRow__td">{user.username}</td>
+        {/*
+          FIXME: user profiles info
+          <td className="UserListRow__td">{user.fullName}</td>
```

Comment:
Please decomment and place mock (hardcoded) data, to have at least the visual implementation